OpenRA.Mods.Common/HitShapes/Capsule.cs

```diff
@@ -92,7 +92,7 @@ public WDist DistanceFromEdge(WPos pos, WPos origin, WRot orientation)
 			return DistanceFromEdge((pos - new WPos(origin.X, origin.Y, pos.Z)).Rotate(-orientation));
 		}
 
-		IEnumerable<IRenderable> IHitShape.RenderDebugOverlay(WorldRenderer wr, WPos origin, WRot orientation)
+		IEnumerable<IRenderable> IHitShape.RenderDebugOverlay(WorldRenderer wr, WPos origin, WRot orientation, bool disabled)
```

Comment:
Question rather than necessarily a change request: would it make more sense to do `IEnumerable<IRenderable> IHitShape.RenderDebugOverlay(HitShape hs, WorldRenderer wr, WPos origin, WRot orientation)` and then check `hs.IsTraitDisabled` as future-proofing against wanting to access any other HitShape state in the future?

Reply:
Can't hurt, I guess. Updated.